shared/middleware/helpers/bucket_helper.rb

```diff
@@ -24,7 +24,7 @@ def category_from_file_type(extension)
     mime_type = Sinatra::Base.mime_type(extension)
     if mime_type == 'application/pdf'
       'pdf'
-    elsif ['.doc', '.docx'].include? extension
```

Comment:
It appears that this method specifies `doc` explicitly because the mime types for .doc and .docx files (as well as .pdf) start with `'application/'...` and can't be generally parsed. 

@joshlory this sound right?